earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -67,10 +65,8 @@ sudo yum install git-lfs
 
 ### CentOS 7 and RHEL 7
 
-To obtain GCC 4.8 on RHEL, you must install the "Development Tools" from RHSM: 
 ```
-sudo yum --setopt=group_package_types=mandatory,default,optional groupinstall "Development Tools"
-sudo yum install -y devtoolset-2-toolchain 
+yum install ant bzip2 doxygen gcc-c++ patch python-argparse python-setuptools swig tar
```

Comment:
I'm working with a minimal install of RHEL 7 and got this:
```
rsync -rltpvu /root/earthenterprise/earth_enterprise/src/NATIVE-REL-x86_64/third_party/libjs/install/lib/ /root/earthenterprise/earth_enterprise/src/NATIVE-REL-x86_64/lib/
sh: rsync: command not found
```
So, that's a package that needs to be added